### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HansOlsson**
Add link to project bylaws.
Note that the bylaws must also be made public - I can easily do that, but wanted to have some feedback before doing both of these things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <img src="https://github.com/modelica/MA-Logos/raw/master/HighRes/Modelica_Language.svg?sanitize=true" width="250px"/>
 
 # ModelicaSpecification
-This repository contains the Modelica Language Specification, hosted at https://github.com/modelica/ModelicaSpecification. Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects).
+This repository contains the Modelica Language Specification, hosted at https://github.com/modelica/ModelicaSpecification.
+Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md)
```
Comment (HansOlsson) [inline README.md:5]:
```suggestion
Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md).
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # ModelicaSpecification
 This repository contains the Modelica Language Specification, hosted at https://github.com/modelica/ModelicaSpecification.
-Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md)
+Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md).
```
Comment (henrikt-ma) [inline README.md:5]: How about taking the opportunity to get the correct MAP-Lang casing?
```suggestion
Development is organized within the [Modelica Association Project Language (MAP-Lang)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md).
```

Reference: https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md#abbreviated-name-of-project

**Review by henrikt-ma — APPROVED** [review]
I had a suggestion, but the PR is good regardless.

File: README.md (hunk 2 above)
Comment (henrikt-ma) [reply README.md:5]: By the way, @sjoelund, here's another place where the all upper case LANG is wrong:
https://modelica.org/projects.html

Comment (HansOlsson) [reply README.md]: Yes. Did that - together renaming the project.

Comment (henrikt-ma) [reply README.md]: Nice! Then https://modelica.org/projects.html is the only place I'm aware of that remains to be fixed.

**Pull request merged**